### ISSUE TITLE
demv.fit_transform() Not changing the dataset 

### ISSUE THREAD

**Issue by pradeepdev-1995** [giordanoDaloisio/demv2022#3]
I have printed the multiclassification dataset before calling the demv.fit_transform()  and also printed the transformed dataset that returns the fit_transform() function. I could observe that the dataset is not changing before and after, only the order of rows in the dataset is changing. All the other data content remains the same other than this order change. is it a desired behavior? Please put the explanation of this.

**Comment by giordanoDaloisio** [comment]
Hi, have you tried to compute the fairness metrics before and after the application of DEMV?
Anyway, from our experiments, we noticed that the size of the dataset does not change too much

**Comment by pradeepdev-1995** [comment]
Yes metric values are changing before and after mitigation. But all the other data content remains the same other than this order change in the transformed dataset. How it is possible @giordanoDaloisio ?

**Comment by giordanoDaloisio** [comment]
If the metrics are changing, it means that the algorithm is working. The algorithm works by rebalancing all the sensitive groups by deleting or duplicating items. Please refer to the linked papers for more clarification.
